### ISSUE TITLE
Not able to format when there's a self-closing tag in Angular

### ISSUE THREAD

**Issue by HymanZHAN** [prettier/prettier-vscode#2891]
### Summary

The VS Code plugin refuses to format Angular templates with self-closing tags.

I believe updating the Prettier version used by the extension will fix this issue. As per mentioned here: https://github.com/prettier/prettier/issues/14168, this issue is already fixed in Prettier 2.8.3.

### Github Repository to Reproduce Issue

https://stackblitz.com/edit/angular-2hxq5d?file=src/app.component.html

### Steps To Reproduce:

1. Clone the project
2. Use the Prettier extension to format `src/app.component.html`.
3. Comment line 5 of `src/app.component.html` and try formatting again.

or

1. Try to run the Prettier formatter on `src/app.component.html` in Stackblitz and see that it doesn't change.

### Expected result

The file should be formatted.

### Actual result

The file is not formatted.

### Additional information

The template file looks like this:
![image](https://user-images.githubusercontent.com/26806995/222970699-7cf48497-914a-4166-b2aa-8d8a369565c6.png)

**VS Code Version**:

Version: 1.76.0
Commit: 92da9481c0904c6adfe372c12da3b7748d74bdcb
Date: 2023-03-01T10:25:16.105Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Linux x64 6.1.14-200.fc37.x86_64
Sandboxed: Yes

**Prettier Extension Version**:
9.10.4

**OS and version**:
Fedora 36

## Prettier Log Output

```
["INFO" - 11:48:14 PM] Extension Name: esbenp.prettier-vscode.
["INFO" - 11:48:14 PM] Extension Version: 9.10.4.
["INFO" - 11:49:05 PM] Formatting file:///home/xzhan/Downloads/angular-ljcgym/src/app.component.html
["INFO" - 11:49:05 PM] Using ignore file (if present) at /home/xzhan/Downloads/angular-ljcgym/.prettierignore
["INFO" - 11:49:05 PM] File Info:
{
  "ignored": false,
  "inferredParser": "angular"
}
["INFO" - 11:49:05 PM] No local configuration (i.e. .prettierrc or .editorconfig) detected, falling back to VS Code configuration
["INFO" - 11:49:05 PM] Prettier Options:
{
  "arrowParens": "always",
  "bracketSpacing": true,
  "endOfLine": "lf",
  "htmlWhitespaceSensitivity": "css",
  "insertPragma": false,
  "singleAttributePerLine": false,
  "bracketSameLine": false,
  "jsxBracketSameLine": false,
  "jsxSingleQuote": false,
  "printWidth": 80,
  "proseWrap": "preserve",
  "quoteProps": "as-needed",
  "requirePragma": false,
  "semi": true,
  "singleQuote": false,
  "tabWidth": 2,
  "trailingComma": "es5",
  "useTabs": false,
  "vueIndentScriptAndStyle": false,
  "filepath": "/home/xzhan/Downloads/angular-ljcgym/src/app.component.html",
  "parser": "angular"
}
["ERROR" - 11:49:05 PM] Error formatting document.
["ERROR" - 11:49:05 PM] Only void and foreign elements can be self closed "child-cmp" (5:5)
  3 |       Learn more about Angular 
  4 |     </a>
> 5 |     <child-cmp />
    |     ^^^^^^^^^^
SyntaxError: Only void and foreign elements can be self closed "child-cmp" (5:5)
  3 |       Learn more about Angular 
  4 |     </a>
> 5 |     <child-cmp />
    |     ^^^^^^^^^^
	at u (/home/xzhan/.vscode/extensions/esbenp.prettier-vscode-9.10.4/node_modules/prettier/parser-html.js:4:2233)
	at Zl (/home/xzhan/.vscode/extensions/esbenp.prettier-vscode-9.10.4/node_modules/prettier/parser-html.js:36:6398)
	at gs (/home/xzhan/.vscode/extensions/esbenp.prettier-vscode-9.10.4/node_modules/prettier/parser-html.js:36:7902)
	at Object.parse (/home/xzhan/.vscode/extensions/esbenp.prettier-vscode-9.10.4/node_modules/prettier/parser-html.js:36:8674)
	at Object.parse (/home/xzhan/.vscode/extensions/esbenp.prettier-vscode-9.10.4/node_modules/prettier/index.js:7331:23)
	at coreFormat (/home/xzhan/.vscode/extensions/esbenp.prettier-vscode-9.10.4/node_modules/prettier/index.js:8645:18)
	at formatWithCursor2 (/home/xzhan/.vscode/extensions/esbenp.prettier-vscode-9.10.4/node_modules/prettier/index.js:8837:18)
	at /home/xzhan/.vscode/extensions/esbenp.prettier-vscode-9.10.4/node_modules/prettier/index.js:37832:12
	at Object.format (/home/xzhan/.vscode/extensions/esbenp.prettier-vscode-9.10.4/node_modules/prettier/index.js:37846:12)
	at t.default.format (/home/xzhan/.vscode/extensions/esbenp.prettier-vscode-9.10.4/dist/extension.js:1:14731)
	at async t.PrettierEditProvider.provideEdits (/home/xzhan/.vscode/extensions/esbenp.prettier-vscode-9.10.4/dist/extension.js:1:11417)
	at async q.provideDocumentFormattingEdits (/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:98:39883)
["INFO" - 11:49:05 PM] Formatting completed in 25ms.
```

**Comment by lucassvasconcelos** [comment]
Same here with font-awesome tags and my custom components:

<fa-icon icon="xmark" class="fa-lg" />

["ERROR" - 17:59:56] Only void and foreign elements can be self closed "fa-icon" (5:4)
> 5 | 			<fa-icon icon="xmark" class="fa-lg" />
    | 			^^^^^^^^

**Comment by MQL9011** [comment]
I have encountered the same problem.

**Comment by MQL9011** [comment]
> This should be fixed now as the extension ships with 2.8.7. Let me know if you still see the issue.

I have encountered the same problem.
